website/client-old/manifest.json

```diff
@@ -30,9 +30,12 @@
 
```

Comment:
Actually, I do have 3 more PRs before this will go live. Do you want me to remove and add the dependencies throughout each PR? It does seem a little odd :P